### ISSUE TITLE
New SD keys won't show lines, but copies of example keys will

### ISSUE THREAD

**Issue by sanderdatema** [markcellis/X-KeyPad#40]
I copied the APR key from the examples to a new key. I changed the key so the text of the key is changed when another dref is on or off. Just as it should be.

Now I create the exact same key from scratch. But the text of the key is not updated at all. Doesn't matter what the condition is set to. Background images do work however.

When looking at the json file, the settings for both keys are exactly the same (apart from the identifiers of course).

So I'm thinking something in the plugin is wrong? Something that is stored in memory?

**Comment by markcellis** [comment]
I’ll try to reproduce it. Can post the broken json file and tell me which key is the from scratch key that is not working? Are you sure you don’t have one of the logic test set to always true?

 

Mark

 

From: Sander Datema ***@***.*** 
Sent: Wednesday, March 15, 2023 4:31 PM
To: markcellis/X-KeyPad
Cc: Subscribed
Subject: [markcellis/X-KeyPad] New SD keys won't show lines, but copies of example keys will (Issue #40)

 

I copied the APR key from the examples to a new key. I changed the key so the text of the key is changed when another dref is on or off. Just as it should be.

Now I create the exact same key from scratch. But the text of the key is not updated at all. Doesn't matter what the condition is set to. Background images do work however.

When looking at the json file, the settings for both keys are exactly the same (apart from the identifiers of course).

So I'm thinking something in the plugin is wrong? Something that is stored in memory?
